staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go

```diff
@@ -99,15 +99,16 @@ type Interface interface {
 	Describe() string
 }
 
-// Manufacture will create a lock of a given type according to the input parameters
-func New(lockType string, ns string, name string, coreClient corev1.CoreV1Interface, coordinationClient coordinationv1.CoordinationV1Interface, rlc ResourceLockConfig) (Interface, error) {
+// New will create a lock of a given type according to the input parameters
```

Comment:
supernit: Match case of function name:

```suggestion
// new will create a lock of a given type according to the input parameters
```